docs/docs/faq.md

```diff
@@ -0,0 +1,60 @@
+# Frequently asked questions
+
+### Why is the `NEBARI_KUBECONFIG` file in `/tmp`?
+
+Nebari regenerates this file on every run. Yes, it will be removed by the operating system during its cleanup process, but running the `qhub deploy` command again as Nebari will update/create a `NEBARI_KUBECONFIG` file for you.
+
+### How are Nebari conda user environments created? Who creates them?
+
+The short answer: there are currently *two* ways of creating environments, as we are in the process of migrating Nebari to Conda-Store, and so which way depends on your use-case.
+
+The longer answer: for global environments, you can specify the environment in `qhub_config.yml` and it will be made available for all users and services (e.g., CDSDashboards). Creating the environments in Conda-Store, by comparison, will provide more granular control over certain settings and permissions.
+
+As Nebari and Conda-Store mature, the intent is to migrate exclusively to Conda-Store for environment creation and management.
+
+### What if the user requires package `X` and it's not available in the environment?
+
+You can simply add the package to the `qhub_config.yml`. If the user doesn't have access to the deployment repo, they'll need to contact their Nebari admin to
+include the required package.
+
+### What's included in the user's environment if a user wants to use Dask?
+
+There are drop-in replacements for `distributed`, `dask`, and `dask-gateway` with the correct pinned versions available via the [QHub Dask metapackage](https://github.com/conda-forge/qhub-dask-feedstock). Example: `qhub-dask==||QHUB_VERSION||`.
+
+### How can a user install a package locally? Is it available to the user's Dask workers?
+
+:::caution
+
+We *strongly recommend* installing packages by adding them through the Conda-Store UI. In an emergency situation, the following approach may be used.
+
+:::
+
+If the user is using a `setuptools` package, they can install it into their local user environment by:
+
+```shell
+pip install --no-build-isolation --user -e .
+```
+
+If they're using a `flit` package, they can install with
+
+```shell
+flit install -s
+```
+
+These aren't available to the Dask workers. See our [Dask tutorial](tutorials/using_dask.md) for more information.
+
+### How can users use .bashrc on Nebari?
```

Comment:
```suggestion
### How can users use `.bashrc` on Nebari?
```

Reply:
Also - not sure if the question is more like "can customise their .bashrc" rather than "can use" - the latter is always available on Linux/UNIX based systems, but the most frequent need to interact with these is to customise or add items to improve the developer workflow/experience

Reply:
It wasn't clear to me either; my best interpretation is "can I have my `.bashrc` on Nebari do the same things my `.bashrc` on my linux box does", though I think phrasing it as "customize" would be clearer.